packages/ui/utils/music.ts

```
@@ -0,0 +1,310 @@
+      slug: song.artists[0].slug,
+    }
+  }
+
+  if (song.artistRels && song.artistRels.length > 0) {
+    return {
+      name: song.artistRels[0].artist_name,
+      slug: song.artistRels[0].artist_slug,
+    }
+  }
+
+  if (song.releaseGroup && song.releaseGroup.artists && song.releaseGroup.artists.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)